lib/account/signup.dart

```diff
@@ -1,14 +1,376 @@
 // Essential Flutter packages
+import 'dart:ui';
+import 'package:atlas/inc/include.dart';
+import 'package:atlas/landing.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
 
+import '../travel/travel.dart' show Travel;
+import 'package:flutter/services.dart';
+
+Include _include = new Include();
+
 class Signup extends StatefulWidget {
   @override
   _SignupState createState() => _SignupState();
 }
 
 class _SignupState extends State<Signup> {
+  bool agree = false;
+  TextEditingController _usernameController = new TextEditingController();
+  TextEditingController _passwordController = new TextEditingController();
+  TextEditingController _emailController = new TextEditingController();
+  TextEditingController _repeatPasswordController = new TextEditingController();
+  void clearTextField(TextEditingController textField) {
+    setState(() {
+      textField.clear();
+    });
+  }
+
   @override
   Widget build(BuildContext context) {
-    return Scaffold(body: Container(child: Column(children: <Widget>[])));
+    SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle(
+        statusBarColor: Colors.transparent,
+        statusBarBrightness: Brightness.dark,
+        systemNavigationBarColor: Color.fromRGBO(193, 55, 111, 1),
+        systemNavigationBarIconBrightness: Brightness.light));
```

Comment:
Make a method in include.dart

Reply:
> Good Job 
> 
> There was a **conflicts** in "include.dart, sun.png".
> 
> * always make sure you have the latest version of release-long-live before PR. [Term 2](https://github.com/PresKhaled/atlas/projects/1#card-36751503)
> 
> For "Terms & Privacy", it's a link, you can use [url_launcher package](https://pub.dev/packages/url_launcher) for this.

I was take latest version , but I don't know what's the problem

Reply:
> Make a method in include.dart

sure , and will create confirmation modal